pkg/parse/parse.go

```diff
@@ -195,8 +209,8 @@ func ParseVolume(volume string) (specs.Mount, error) {
 	return mount, nil
 }
 
-// ParseVolumes validates the host and container paths passed in to the --volume flag
-func ParseVolumes(volumes []string) error {
+// parseVolumes validates the host and container paths passed in to the --volume flag
+func parseVolumes(volumes []string) error {
```

Comment:
Why are you making these private?  Doesn't Podman use these?

Reply:
No. this is not from libpod. it is only used in buildah parse.go

Reply:
I am saying the Podman is vendoring this code into its code base and uses this function to parse its objects.

Reply:
No, I don't think podman needs this function.

Reply:
Leave this exported for now, once we have patches podman to support Overlay Mounts, if we don't use these interfaces, then you can unexport them. Otherwise I will have to open a PR to reverse this patch to make them exportable again.